lib/src/main/java/uk/ac/manchester/beehivespirvtoolkit/lib/tests/TestRunnerAssembler.java

```diff
@@ -148,7 +148,7 @@ public static void testEmptyKernel() throws InvalidSPIRVModuleException {
         blockScope.add(new SPIRVOpReturn());
         functionScope.add(new SPIRVOpFunctionEnd());
 
-        writeModuleToFile(module,"/tmp/testSPIRV.spv");
+        writeModuleToFile(module,"testSPIRV.spv");
```

Comment:
I guess this has been updated in order to ensure compatibility between Linux/Windows file systems, right?